infra/src/main/kotlin/fi/fta/geoviite/infra/ratko/RatkoRouteNumberService.kt

```diff
@@ -28,12 +34,18 @@ class RatkoRouteNumberService @Autowired constructor(
     private val logger: Logger = LoggerFactory.getLogger(this::class.java)
 
     fun pushTrackNumberChangesToRatko(
+        branch: LayoutBranch,
         publishedTrackNumbers: Collection<PublishedTrackNumber>,
         publicationTime: Instant,
     ): List<Oid<TrackLayoutTrackNumber>> {
-        logger.serviceCall("pushTrackNumberChangesToRatko",
-            "publishedTrackNumbers" to publishedTrackNumbers, "publicationTime" to publicationTime
+        logger.serviceCall(
+            "pushTrackNumberChangesToRatko",
+            "branch" to branch,
+            "publishedTrackNumbers" to publishedTrackNumbers,
+            "publicationTime" to publicationTime,
         )
+        // TODO: Design branches not yet supported. We need to ensure that design tracknumbers have own OIDs -> could we rely on tracknumber.branch instead of arg?
```

Comment:
Koska ratkon design-puolta ei ole vielä kunnolla mietitty ja heräsin ajattelemaan tätäkin dilemmaa (pitäisikö branchin olla eksplisiittinen arg vai onko se jotain mikä vain päätellään julkaisusta), päätin jättää tuohon todo:n muistutukseksi. Oikeasti näitä mietitään myöhemmin tuolla: https://extranet.vayla.fi/jira/browse/GVT-2647